cmd/user-agent/src/pages/DIDManagement.vue

```diff
@@ -76,33 +73,83 @@ SPDX-License-Identifier: Apache-2.0
                 const keyset= await window.$aries.kms.createKeySet()
                 const recoveryKeyset= await window.$aries.kms.createKeySet()
 
-                // create did request
-                // TODO Support P-256 key type
-                const createDIDRequest = {
-                    "publicKeys":[{
-                        "id":"key-1",
-                        "type":"JwsVerificationKey2020",
-                        "value":keyset.signaturePublicKey,
-                        "encoding":"Jwk",
-                        "keyType":this.selectType,
-                        "usage":["ops","general"]
-                    }, {
-                        "id":"key-recovery",
-                        "type":"JwsVerificationKey2020",
-                        "value":recoveryKeyset.signaturePublicKey,
-                        "encoding":"Jwk",
-                        "keyType":this.selectType,
-                        "recovery":true
-                    }
-                    ]
-                };
+               let createDIDRequest ={}
```

Comment:
can change if else to this                                                                                                                                                                     
               let keyValue ={}
                let recoveryKeyValue ={}
                if (this.selectType == "Ed25519"){
                    keyValue=keyset.signaturePublicKey
                    recoveryKeyValue=recoveryKeyset.signaturePublicKey
                }
                const createDIDRequest = {
                    "publicKeys":[{
                        "id":"key-1",
                        "type":"JwsVerificationKey2020",
                        "value":keyValue,
                        "encoding":"Jwk",
                        "keyType":this.selectType,
                        "usage":["ops","general"]
                    }, {
                        "id":"key-recovery",
                        "type":"JwsVerificationKey2020",
                        "value":recoveryKeyValue,
                        "encoding":"Jwk",
                        "keyType":this.selectType,
                        "recovery":true
                    }
                    ]
                };

Reply:
done